pep/app/pods/components/page/sidebar/widgets/template.hbs

```diff
@@ -1,20 +1,22 @@
-<div class="px-2 pb-1 d-flex justify-content-between align-items-center" ...attributes>
-    <h5 class="m-0">{{t "relatedInfo.title"}}</h5>
-    <Button
-        @type="link"
-        @action={{this.toggleAll}}
-        @label={{t (if this.showCloseAll "relatedInfo.batchClose" "relatedInfo.batchOpen")}}
-        @icon={{if this.showCloseAll "chevron-up" "chevron-down"}}
-        class="p-0 font-size-secondary"
-    />
-</div>
-<div class="p-2">
-    {{#each @widgets as |widget|}}
-        {{component
-            (concat "page/sidebar/widgets/" widget.widget)
-            toggleIsOpen=this.toggleIsOpen
-            openWidgets=this.openWidgets
-            data=@data
-        }}
-    {{/each}}
-</div>
\ No newline at end of file
+{{#if @widgets}}
```

Comment:
Seems like you have two ways to hide the widgets. `hideWidgets` from above, and this. Do you need both?

Reply:
yeah, so the `{{#if @widgets}}` is for in case in the Admin configs they decide to not show any widgets on the left sidebar, in which case we dont want the "Related Info" and toggle button to show there.

the `@hideWidgets` arg is for the search route's left sidebar, where it needs to hide all the widgets if there is search results (i.e. the "Refine" section is visible).